src/test/java/org/example/stepsDefinitions/LoginFunctionalitiesSteps.java

```diff
@@ -0,0 +1,30 @@
+package org.example.stepsDefinitions;
+
+import io.cucumber.java.en.*;
+
+
+public class LoginFunctionalitiesSteps {
+	@Given("the user has navigated to the login page")
+	public void theUserHasNavigatedToTheLoginPage() {
+
+	}
+
+	@When("the user enters the correct username and password")
+	public void theUserEntersTheCorrectUsernameAndPassword() {
+
+	}
+
+	@Then("the user should be redirected to the home page")
+	public void theUserShouldBeRedirectedToTheHomePage() {
+		
+	}
+
+	@When("the user enters an incorrect username or password")
+	public void theUserEntersAnIncorrectUsernameOrPassword() {
+		
+	}
+
+	@Then("the user should see an error message indicating invalid login credentials")
+	public void theUserShouldSeeAnErrorMessageIndicatingInvalidLoginCredentials() {
+	}
```

Comment:
Could you please provide some steps implementation.

Reply:
Implementation Added I forgot to push last changes, sorry for that